src/main/java/dk/cngroup/lentils/service/convertors/TeamConverter.java

```diff
@@ -0,0 +1,23 @@
+package dk.cngroup.lentils.service.convertors;
+
+import dk.cngroup.lentils.entity.Team;
+import dk.cngroup.lentils.entity.formEntity.TeamDTO;
+import org.springframework.stereotype.Service;
+
+@Service
+public class TeamConverter {
+
+    public TeamDTO fromEntity(final Team team) {
+        final TeamDTO dto = new TeamDTO();
+
+        dto.setName(team.getName());
+        dto.setNumOfMembers(team.getNumOfMembers());
+
+        return dto;
+    }
+
+    public void toEntity(final TeamDTO dto, final Team team) {
```

Comment:
Proc ma metoda toEntity parametr `team` (tj. to co se vraci) a metoda fromEntity nema? toto by chtelo sjednotit a vice se mi libi varianta, kde se ten vystupni objekt neposila jako parametr, ale vytvori se az v teto metode.

Reply:
On je ten druhý parametr výstupní. Pro případ kde se setuje existující entita při updatu a nevytváří se celá znovu. Nebo myslíš, že by to šlo uděla ještě líp?